metricbeat/helper/prometheus/prometheus.go

```diff
@@ -83,6 +84,7 @@ func (p *prometheus) GetFamilies() ([]*dto.MetricFamily, error) {
 			if err == io.EOF {
 				break
 			}
+			return nil, errors.Wrap(err, "Decode MetricFamily failed")
```

Comment:
Error message should start with lowercase.
```suggestion
			return nil, errors.Wrap(err, "decoding of metric family failed")
```